impl/src/main/java/org/jboss/weld/xml/XmlSchema.java

```diff
@@ -20,7 +20,7 @@
 
 public enum XmlSchema {
 
-    CDI10("beans_1_0.xsd", BeanManager.class.getClassLoader()), CDI11("beans_1_1.xsd", BeanManager.class.getClassLoader()), WELD11("weld_1_1.xsd", BeansXmlParser.class.getClassLoader());
+    CDI20("beans_2_0.xsd", BeanManager.class.getClassLoader()), WELD11("weld_1_1.xsd", BeansXmlParser.class.getClassLoader());
```

Comment:
We can't remove `beans_1_0.xsd` -> it has a different namespace and we would like to support old versions of `beans.xml`.